### DISCUSSION TITLE
Logging to syslog

### DISCUSSION THREAD

**Discussion by trsqr** [wmbusmeters/wmbusmeters#678] [Ideas]
I have a Raspberry Pi and want to configure wmbusmeters to log to syslog instead of file in order to preserve the flash (syslog is configured to do remote syslogging to the NAS). As far as I see it, wmbusmeters is only logging to a file. Would it be possible to add syslog logging as well?

**Comment by BIBOLV** [comment]
Try setting `logfile=syslog` and see if it works.

**Comment by weetmuts** [comment]
Thanks! Just pushed a fix for this. --logfile=syslog now writes to syslog.
